src/pages/Resources/index.tsx

```diff
@@ -53,12 +53,12 @@ const linkProps: AppLinkProps[] = [
   }
 ];
 
-const AppsPageComponent: React.SFC = () => (
+const ResourcesPageComponent: React.SFC = () => (
   <React.Fragment>
     <div className="content">
       <div className="apps">
```

Comment:
Change these classnames as well.
```suggestion
       <div className="resources">
```